.golangci.yml

```diff
@@ -46,6 +46,12 @@ linters:
     - structcheck
     - wastedassign
 
+    # review later before merging to master
```

Comment:
You wont use those linters?

Reply:
Nope, I won't. I will still review it in the future. Therefore, the comment can remain